webpack.config.js

```diff
@@ -41,16 +45,40 @@ const config = {
             }
           }
         ]
+      },
+      {
+        test: /opencascade\.wasm\.wasm$/,
+        type: "javascript/auto",
+        loader: "file-loader"
       }
     ]
   },
+  node: {
+    fs: "empty"
+  },
   plugins: [
     new HtmlWebpackPlugin({
       appMountId: "app",
       filename: "index.html",
       template: "index.html"
     }),
-    new MonacoWebpackPlugin()
+    new MonacoWebpackPlugin(),
+    new CopyPlugin({
+      patterns: [
+        {
+          from: "js/StandardLibraryIntellisense.ts",
+          to: "js/StandardLibraryIntellisense.ts"
```

Comment:
This copies files from the repo and puts them into `dist` so they're available for the app to grab.